src/components/List/List.tsx

```diff
@@ -1,4 +1,4 @@
-import React, {PureComponent} from 'react';
+import React, {NamedExoticComponent} from 'react';
```

Comment:
Were you planning on doing something with this?

Reply:
ooh whoops. That was a hangover from a slightly different trick to tempt the types into playing nice that wasn't needed. Removed!